base_geoengine/__manifest__.py

```diff
@@ -2,31 +2,35 @@
 # Copyright 2011-2015 Nicolas Bessi (Camptocamp SA)
 # Copyright 2016 Yannick Vaucher (Camptocamp SA)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
-{'name': 'Geospatial support for Odoo',
- 'version': '10.0.1.0.0',
- 'category': 'GeoBI',
- 'author': "Camptocamp,ACSONE SA/NV,Odoo Community Association (OCA)",
- 'license': 'AGPL-3',
- 'website': 'http://openerp.camptocamp.com',
- 'depends': [
-     'base',
-     'web'
- ],
- 'init_xml': [],
- 'data': [
-     'security/data.xml',
-     'views/base_geoengine_view.xml',
-     'geo_ir/ir_model_view.xml',
-     'geo_view/ir_view_view.xml',
-     'geo_view/geo_raster_layer_view.xml',
-     'geo_view/geo_vector_layer_view.xml',
-     'security/ir.model.access.csv',
- ],
- 'external_dependencies': {
-     'python': ['shapely',
-                'geojson'],
- },
- 'qweb': ["static/src/xml/geoengine.xml"],
- 'installable': True,
- 'pre_init_hook': 'init_postgis',
- }
+{
+    'name': 'Geospatial support for Odoo',
+    'version': '10.0.1.0.0',
+    'category': 'GeoBI',
+    'author': "Camptocamp,ACSONE SA/NV,Odoo Community Association (OCA)",
+    'license': 'AGPL-3',
+    'website': 'http://openerp.camptocamp.com',
+    'depends': [
+        'base',
+        'web'
+    ],
+    'init_xml': [],
+    'data': [
+        'security/data.xml',
+        'views/base_geoengine_view.xml',
+        'geo_ir/ir_model_view.xml',
+        'geo_view/ir_view_view.xml',
+        'geo_view/geo_raster_layer_view.xml',
+        'geo_view/geo_vector_layer_view.xml',
+        'security/ir.model.access.csv',
+    ],
+    'external_dependencies': {
+        'python': [
+            'shapely',
+            'geojson',
+            'simplejson'
+        ],
+    },
+    'qweb': ["static/src/xml/geoengine.xml"],
```

Comment:
Insn't this deprecated in favour of a extending the assets with QWeb XML?

Reply:
Not for qweb I think, unless I missed something. You extend the asset template only to add javascript and css file.

Reply:
* Not deprecated for static QWeb. Even if extending a static QWeb, you would need to include in here except in some special cases where the JS loads it

Reply:
ok